ckanext/push_errors/plugin.py

```diff
@@ -86,3 +96,23 @@ def make_error_log_middleware(self, app, config):
 
     def get_commands(self):
         return [push_errors_commands]
+
+    def get_actions(self):
+        return {
+            'push_errors_enable': self.push_errors_enable,
+            'push_errors_disable': self.push_errors_disable,
+        }
+
+    def push_errors_enable(self, context, data_dict):
+        """Enable the sending of notifications"""
+        cache = get_cache()
+        cache.set('push_errors:enabled', '1')
+        log.info("push-errors: Notifications enabled.")
+        return {'status': 'enabled'}
+
+    def push_errors_disable(self, context, data_dict):
+        """Disable the sending of notifications"""
+        cache = get_cache()
+        cache.set('push_errors:enabled', '0')
+        log.info("push-errors: Notifications disabled.")
+        return {'status': 'disabled'}
```

Comment:
CKAN actions must be at `actions.py` file or `actions` folder

Reply:
https://github.com/unckan/ckanext-push-errors/pull/10/commits/e34dca5959269668ba8e5837234fcd92415d8282